boringssl/Dockerfile

```diff
@@ -18,6 +18,5 @@ FROM ossfuzz/base-libfuzzer
 MAINTAINER mike.aizatsky@gmail.com
 RUN apt-get install -y cmake ninja-build golang
 
-ENV GIT_URL "https://boringssl.googlesource.com/boringssl"
-
+RUN git clone https://boringssl.googlesource.com/boringssl
```

Comment:
@oliverchang yes, I did try to do with CHECKOUT_COMMAND. There are couple reasons why I decided on this approach:

- I want to support binding of /src/mylibrary volume to be able to work with local checkout. This means that the script has to be smart enough to not checkout things over my working copy

- handling of multiple source roots becomes weird. (It is still git, but now you have to have a custom command).

- this way there's absolutely no requirements on the VCS to use. You can even fetch a tarball or do anything you want.

>  It seems a bit weird to me that the image will get a fixed checkout of the repository.

It was for me at first. But image was never supposed to be applicable to all library versions (what if build procedure changes?). I think it is totally reasonable to let it fetch whatever it needs to later build the fuzzer. Actually all the work that is not dependent on current sanitizer flags (ie `CFLAGS` etal) should go into image building time.